src/main/java/Duke.java

```diff
@@ -1,10 +1,373 @@
+import java.util.Locale;
+import java.util.Objects;
+import java.util.Scanner;
+
+
 public class Duke {
+
+    private static final Task[] tasks = new Task[Constants.MAX_NUMBER_OF_ENTRIES];
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        String command;
+        int numberOfTasks = 0;
+        String latestResponse = "";
+        boolean hasAdditionalTask = false;
+
+        greetUser();
+
+        while (numberOfTasks < 100) {
+            command = latestResponse;
+
+
+            if (!hasAdditionalTask) {
+                System.out.println("What can I do for your today?");
+                command = readTask();
+            }
+            boolean hasDuplicateKeywords = checkForDuplicateKeyword(command);
+
+            while (hasDuplicateKeywords == true) {
+                printDuplicateKeywordErrorMessage();
+                command = readTask();
+                hasDuplicateKeywords = checkForDuplicateKeyword(command);
+            }
+            printHorizontalBar();
```

Comment:
I like the usage of a method for the printing of the '_' as it looks much neater.